.github/workflows/applications.yml

```diff
@@ -184,24 +179,111 @@ jobs:
       shell: bash
       run: |
         export VER=$(grep '^version' pact.cabal | sed -e 's/.*: *//')
-        mkdir -p dist-newstyle/artifacts/applications
-        cp dist-newstyle/build/*/ghc-${{ matrix.ghc }}/pact-${VER}/x/pact/build/pact/pact dist-newstyle/artifacts/applications
+        mkdir -p artifacts/pact
+        cp $(cabal list-bin pact) artifacts/pact
+        cp CHANGELOG.md artifacts/pact
+        cp README.md artifacts/pact
+        cp LICENSE artifacts/pact
+        cp pact.cabal artifacts/pact
+        cp cabal.project artifacts/pact
+        cp cabal.project.local artifacts/pact
+        cp cabal.project.freeze artifacts/pact
```

Comment:
Probably don't want `cabal.project.local` in there - that should always be local build info never global build info.

Reply:
It's generated just for the build and included to document how the image was build. It makes the build reproducible.